src/Network/Network/PrivateLinkService/PrivateLinkServiceIpConfiguration/NewAzurePrivateLinkServiceIpConfiguration.cs

```diff
@@ -51,6 +54,7 @@ public class NewAzurePrivateLinkServiceIpConfiguration : NetworkBaseCmdlet
                           "if static allocation is specified.")]
         public string PrivateIpAddress { get; set; }
 
+        [CmdletParameterBreakingChange("PublicIpAddress", ChangeDescription = "Parameter is being deprecated without being replaced")]
```

Comment:
What does a customer do instead of supplying a PublicIPAddres?  Is it generated for them?  Please include information on what the customer does now for the same functionality, or why this is no longer needed.

Reply:
Customer has no needs to do anything because the property 'PublicIpAddress' is totally redundant. No matter what the customer passed in previously,  it would not be used at all.
This property is already been removed from Swagger/SDK.